### ISSUE TITLE
Hyrule Warriors: Age of Calamity AMD Vulkan rendering issues

### ISSUE THREAD

**Issue by Modoh** [yuzu-emu/yuzu#9741]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access 3365

### Description of Issue

Rendering problems on AMD GPU with Vulkan on Windows.   Mostly shows up as black dots on the edges of character models in cutscenes, but also appears in gameplay on the horizon, fading character models, and water.   There are also sometimes colored dots on the horizon or on fading/transparent models.   

Renders properly in Vulkan on a Nvidia card, and in OpenGL on AMD (though OpenGL on AMD tends to crash the driver).

I'm not sure if it's related but the skybox also doesn't render properly on the left side of the screen, including during gameplay. 

![01002b00111a2000_2023-02-06_20-10-18-781](https://user-images.githubusercontent.com/88527532/217124026-274bfec3-5d9c-4781-929e-8a1c75dcd412.png)
![01002b00111a2000_2023-02-06_20-14-44-013](https://user-images.githubusercontent.com/88527532/217124041-a0b67bce-7792-4d34-bfef-3bb5be46379e.png)

### Expected Behavior

Characters shouldn't look like they're covered with space ants, and water shouldn't be a black void. 

### Reproduction Steps

Load up the game on Windows with Vulkan selected on an AMD GPU, the issues occur on the title screen and basically every non-prerendered cutscene.   

Problem occurs on both High and Extreme GPU accuracy and isn't affected by rendering resolution.

### Log File

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/10670597/yuzu_log.txt.old.txt)

### System Configuration

CPU: Intel 9700k
GPU/Driver: AMD 7900XTX 23.1.1
RAM: 32GB
OS: Windows 10 22H2

**Comment by Modoh** [comment]
That's fair enough if it's only broken on RDNA3.   I'm afraid I don't have an older AMD GPU to test with, all of my older stuff is NVidia.

**Comment by goldenx86** [comment]
Yeah, RDNA2 and older is fine. I'm waiting for that driver merge to start nagging AMD.

**Comment by Moonlacer** [comment]
Hello all, I will be taking a look at RDNA3 issues and if I can confirm they are RDNA3 specific, I will report them to AMD

**Comment by IRustyI** [comment]
Unfortunately, AMD's latest driver 23.5.1 does not resolve the Vulkan rendering issues shown here. I cleared the pipeline cache and used the same settings as when I presented the screenshots above, in addition to changing the GPU Accuracy between low, high, and extreme.

**Comment by Moonlacer** [comment]
> Unfortunately, AMD's latest driver 23.5.1 does not resolve the Vulkan rendering issues shown here. I cleared the pipeline cache and used the same settings as when I presented the screenshots above, in addition to changing the GPU Accuracy between low, high, and extreme.

Yes, I will be the first to notify people should the issue be resolved, so stay tuned 😉
